cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.cpp

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "UrdfBotPawn.h"
+
+#include <Camera/CameraComponent.h>
+#include <Components/InputComponent.h>
+
+#include "CoreUtils/Config.h"
+#include "UrdfRobotComponent.h"
+
+AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer): APawn(object_initializer)
+{
+    // setup robot
+    robot_component_ = CreateDefaultSubobject<UUrdfRobotComponent>(TEXT("RobotComponent"));
+    RootComponent = Cast<USceneComponent>(robot_component_->root_link_component_);
```

Comment:
No need to cast here, because `UUrdfRobotComponent` is a `USceneComponent`. If there was a need to cast, our convention is to use `dynamic_cast`.

Reply:
fixed.